src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -370,22 +370,31 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 					let findModel = (editor.notebookParams.input as NotebookInput).notebookFindModel;
 					if (findModel?.findMatches?.length > 0) {
 						let searchString = findModel.findExpression;
-						mark.mark(searchString, {
+						markDoc.mark(searchString, {
 							className: 'rangeHighlight'
 						});
 						elementContainingText.scrollIntoView({ behavior: 'smooth' });
 					}
 				}
+				mark.markRanges([{
+					start: range.startColumn - 1, //subtracting 1 since markdown html is 0 indexed.
+					length: range.endColumn - range.startColumn
+				}], {
+					className: 'rangeSpecifcHighlight'
```

Comment:
typo here - `rangeSpecificHighlight` (make sure to update the other references too)

I'd suggest making this into a const in the file to make this easier and safer.